x-pack/plugins/observability/server/assets/enrich_policies/slo_summary_enrich_policy.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SO_SLO_TYPE } from '../../saved_objects';
+import { SLO_SUMMARY_ENRICH_POLICY_NAME } from '../constants';
+
+export const getSLOSummaryEnrichPolicy = () => ({
+  name: SLO_SUMMARY_ENRICH_POLICY_NAME,
+  match: {
+    indices: '.kibana',
+    match_field: 'slo.id',
+    enrich_fields: [
+      'slo.name',
+      'slo.description',
+      'slo.tags',
+      'slo.indicator.type',
+      'slo.objective.target',
```

Comment:
@kdelemme I think we should add  `slo.objective.timesliceTarget` and `slo.ojective.timesliceWindow` as well. Unless there is a reason why we didn't add these here. 

Also what about `settings`? Should this field be included to the `enrich_fields`? I was looking at these [tests](https://github.com/elastic/kibana/pull/169993/files#diff-db9da77d28e0ff2e63dbe16896c5082e03cf5cfff7fe791594ac53a0b0f5ca80R160) regarding bumping the revision or not during update and I want to be sure we didn't forget to include all required fields to the enrich fields.

Reply:
We haven't added these fields because we don't search with them. I don't know if we should do it now or not